src/shared/components/GeneSelectionBox/OQLTextArea.tsx

```diff
@@ -194,7 +194,8 @@ export default class OQLTextArea extends React.Component<
                 break;
         }
         this.geneQuery
-            ? classNames.push(styles.notEmpty)
+            ? // TODO no such style called `notEmpty`
+              classNames.push(/* styles.notEmpty */ '')
```

Comment:
get rid of this altogether then?

Reply:
most likely we will end up removing all the invalid references before merging the PR, but keeping them for now just in case someone wants to comment.